README.md

```diff
@@ -1 +1,515 @@
+
+## 4.1 Fields
+
+Recall UCAN Invocation Tasks:
+
+| Field    | Type             | Description                                    | Required | Default |
+|----------|------------------|------------------------------------------------|----------|---------|
+| `with`   | `URI`            |                                                | Yes      |         |
+| `do`     | `Ability`        |                                                | Yes      |         |
+| `inputs` | `Any`            |                                                | Yes      |         |
+| `meta`   | `{String : Any}` | Fields that will be ignored during memoization | No       | `{}`    |
+
+An OPTIONAL IPVM `Config` MAY be included at the `meta['ipvm/config']` path. The `meta` field SHOULD not captured as part of task memoization, so this information will be omitted from the distributed invocation table. If included, the `Config` MUST set the IPVM configuration for this Task, overwriting any of the fields on the envelope's top-level `defaults` field, or system-wide defaults.
+
+## 4.3 JSON Examples
+
+``` js
+{
+  "some-wasm": {
+    "with": "ipfs://bafkreicgincymfzzumcqcrpmquyildzqppa3s2gv3scfy2csaqpgqpp7ke",
+    "do": "wasm/run",
+    "inputs": {
+      "func": "calculate",
+      "args": [
+        1,
+        "hello world",
+        {"c": {"ucan/promise": ["/", "some-other-action"]}},
+        {"a": 1, "b": 2, "c": 3}
+      ]
+    },
+    "ipvm/config": {
+      "v": "0.1.0",
+      "secret": false,
+      "check": {
+        "optimistic": 17, 
+        "referee": "did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4"
+      }
+    }
+  }
+}
+```
+
+``` json
+{
+  "set-dns": {
+    "with": "dns://example.com?TYPE=TXT",
+    "do": "crud/update",
+    "inputs": { 
+      "value": "hello world"
+    },
+    "meta": {
+      "ipvm/config": {
+        "secret": false
+        "timeout": [500, "milli", "seconds"],
+        "verification": "attestation"
+      }
+    }
+  }
+}
+```
+
+``` json
+{
+  "with": "ipfs://bafkreidvq3uqoxcxr44q5qhgdk5zk6jvziipyxguirqa6tkh5z5wtpesva",
+  "can": "docker/run",
+  "inputs": {
+    "entrypoint": ["/", "home"],
+    "mounts": {"mnt": "/"},
+    "outputs": {"tmp": "/tmp"},
+    "env":  {"$HELLO": "world"},
+    "workdir":  "/work"
+  },
+  "meta": {
+    "ipvm/config": {
+      "secret": false,
+      "check": {
+        "optimistic": 2, 
+        "referee": "did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4"
+      }
+    }
+  }
+}
+```
+
+``` json
+{
+  "with": {"ucan/promise": ["/", "previous_action"]},
+  "can": "docker/run",
+  "inputs": {
+    "entrypoint": ["/", "home"],
+    "mounts": {"mnt": "/"},
+    "outputs": {"tmp": "/tmp"},
+    "env":  {"$HELLO": "world"},
+    "workdir":  "/work"
+  },
+  "meta": {
+    "ipvm/config": {
+      "disk": "200GB"
+    }
+  }
+}
+```
```

Comment:
From [comments on the old repo](https://github.com/ipvm-wg/spec/pull/8#issuecomment-1332963033), I've included examples of how to both pull a Docker container off the network & run it, as well as how to do this with a pipeline.

Reply:
(fixed `s/can/do/` 😅 )